packages/scipy/meta.yaml

```
@@ -1,60 +1,92 @@
-  # This requires,
-  #    pip install Cython Tempita
-  # on the host for the build
-  url: https://github.com/scipy/scipy/archive/v0.17.1.tar.gz
-  sha256: bf22383d9e2526ffdc53a6c7d3d66dab45a86291ed35d8e281bd1a5d2c34187b
+  url: https://files.pythonhosted.org/packages/61/67/1a654b96309c991762ee9bc39c363fc618076b155fe52d295211cf2536c7/scipy-1.7.3.tar.gz
+  sha256: ab5875facfdef77e0a47d5fd39ea178b58e60e454a4c85aa1e52fcb80db7babf
 
   patches:
-    # these patches can be found as commits in
-    #    https://github.com/rth/scipy/tree/0.17.1-pyodide
-    # on top of the v0.17.1 tag
-    - patches/fix-blas.patch
-    - patches/fix-build-gcc5-a80460.patch
-    - patches/force_malloc.patch
-    - patches/disable_scipy_stats_mvn.patch
-    - patches/skip-fortran-fails-to-link.patch
-    - patches/skip_ellip_harm_2_pyx_ctypes.patch
+    - patches/convert-xerrwv-message-arg-from-int-to-str.patch
+    - patches/disable-blas-detection.patch
+    - patches/fix-fortran-files-minpack.patch
+    - patches/gemm_-no-const.patch
     - patches/make-int-return-values.patch
-    - patches/add-missing-extern.patch
-    - patches/fix-typo-flapack-pyf.patch
+    - patches/remove-cuncsd-dorcsd-sorcsd-zuncsd.patch
+    - patches/remove-mvnun-not-fortran-77-compliant.patch
+    - patches/sasum-returns-double-not-float.patch
+    - patches/skip-fortran-fails-to-link.patch
+    - patches/rename-_page_trend_test.patch
 
 build:
   # set linker and C flags to error on anything to do with function declarations being wrong.
   # In webassembly, any conflicts mean that a randomly selected 50% of calls to the function
   # will fail. Better to fail at compile or link time.
   cflags: |
-    -I $(PYODIDE_ROOT)/packages/numpy/config
+    -I$(PYODIDE_ROOT)/packages/numpy/config
     -I$(PYODIDE_ROOT)/packages/CLAPACK/build/CLAPACK-3.2.1/INCLUDE
+    -I$(PYODIDE_ROOT)/packages/.artifacts/include/
+    -I$(PYODIDE_ROOT)/packages/.artifacts/pythran/
     -DUNDERSCORE_G77
     -Werror=implicit-function-declaration
     -Werror=mismatched-parameter-types
     -Werror=mismatched-return-types
   ldflags: |
     -L$(NUMPY_LIB)
+    -L$(NUMPY_LIB_BASE)/random/lib
     -Wl,--fatal-warnings
+  # IMPORTANT: Other locations important in scipy build process:
+  # There are two files built in the "capture" pass that need patching:
+  #    _blas_subroutines.h, and _cython
+  # Scipy has a bunch of custom logic implemented in
+  # pyodide-build/pyodide_build/_f2c_fixes.py.
   script: |
-    # Some fixes were extracted from patches/make-int-return-values.patch as regex.
+    pip install scipy==1.7.3 pybind11[global] pythran
+    pip install --upgrade pybind11[global] pythran -t $PYODIDE_ROOT/packages/.artifacts/
+    # We get linker errors because the following 36 functions are missing
+    # Copying them from a more recent LAPACK seems to work fine.
+    wget https://github.com/Reference-LAPACK/lapack/archive/refs/tags/v3.10.0.tar.gz
+    tar -xnf v3.10.0.tar.gz
```

```suggestion
    tar xzf v3.10.0.tar.gz
```
? Why the `-n` (seekable) option?  No need to change, I'm just curious.

---

I don't actually know what these things do, I just typed some random letters and it didn't visibly screw up so I moved on to something else.